src/core/CsvParser.h

```diff
@@ -0,0 +1,101 @@
+﻿/*
+ *  Copyright (C) 2016 Enrico Mariotti <enricomariotti@yahoo.it>
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 or (at your option)
+ *  version 3 of the License.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef KEEPASSX_CSVPARSER_H
+#define KEEPASSX_CSVPARSER_H
+
+#include <QFile>
+#include <QBuffer>
+#include <QQueue>
+#include <QTextStream>
+
+typedef QStringList csvrow;
+typedef QList<csvrow> csvtable;
```

Comment:
@seatedscribe I think these need to be PascalCase, since they define types. e.g. `typedef QStringList CsvRow`